app/services/stats/r900_cases_report.rb

```diff
@@ -0,0 +1,40 @@
+require 'csv'
+
+module Stats
+  class R900CasesReport < BaseReport
+
+    CSV_COLUMN_HEADINGS = CSVExporter::CSV_COLUMN_HEADINGS
+
+    def self.title
+      'General cases report'
+    end
```

Comment:
I'd be inclined to just call these `cases` rather than `general cases`... is there any other sort of cases we're distinguishing "general" cases from?